src/App.jsx

```diff
@@ -14,70 +16,67 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [value, setValue] = useState('Jam is selected');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  function getTitle(item) {
+    return value.startsWith(item) ? setValue('No goods selected') : setValue(`${item} is selected`);
+  }
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+  return (
+    <main className="section container">
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
+      <h1 className="title is-flex is-align-items-center">
+        {value}
+        {
+          !value.startsWith('No')
+          && (
             <button
-              data-cy="AddButton"
+              data-cy="ClearButton"
               type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+              className="delete ml-3"
+              onClick={() => setValue('No goods selected')}
+            />
+          )
+        }
+      </h1>
+      <table className="table">
+        <tbody>
+          {
+          goods.map((item) => {
```

Comment:
```suggestion
          {goods.map((item) => {
```
Fix formatting everywhere